packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperToken.sol

```diff
@@ -45,6 +45,11 @@ interface ISuperToken is ISuperfluidToken, TokenInfo, IERC20, IERC777 {
         string calldata s
     ) external;
 
+    /**************************************************************************
+    * Constants
+    *************************************************************************/
+    function SUPERFLUID_NFT_DEPLOYER_LIBRARY_ADDRESS() external view returns (address);
```

Comment:
This should not be there, this is a detail that should not be part of the interface.

Reply:
The one uses this is using "SuperToken" contract, so it's not a problem for the use sites.

Reply:
will remove